docs/contributing.md

```diff
@@ -1,122 +1,39 @@
 # Contributing to PrairieLearn
 
-- [Good blog post on how to contribute via GitHub](http://blog.davidecoppola.com/2016/11/howto-contribute-to-open-source-project-on-github/)
+PrairieLearn is an open-source project and welcomes contributions from anyone. See the [contributing guidelines](https://github.com/PrairieLearn/PrairieLearn/blob/master/CONTRIBUTING.md) to learn more about the contribution process.
 
-## GitHub: distributed model
+## Maintainer guidelines
 
-**Getting started as a collaborator:** Everyone should fork the main repository on GitHub. The full name of this repository is `PrairieLearn/PrairieLearn` (`"organization_name/repo_name"`); when you fork it, GitHub will create a repo named `user_name/PrairieLearn`. If you're not sure how to fork the repo, see [Github's instructions](https://help.github.com/articles/fork-a-repo/).
+A trusted group of core developers are responsible for maintaining the PrairieLearn repository. This responsibility includes upholding code quality, ensuring smooth collaboration, and fostering an open-source community. These guidelines outline the expectations and conventions for maintainers.
 
-Before you go further, you will need to set up your own SSH keys and associate them with your GitHub account (if you haven't already). If you're not sure how to set them up, see [Connecting to GitHub with SSH](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).
+### General principles
 
-Next, clone your forked repository to your local machine and set up the main repository as an "upstream" repository:
+- **Use maintainer privileges wisely**
 
-```sh
-git clone git@github.com:$user_name/PrairieLearn.git
-cd PrairieLearn
-git remote add upstream git@github.com:PrairieLearn/PrairieLearn.git
-```
+  - Do not approve or merge changes you do not fully understand.
+  - If you’d like to understand a PR better, ask questions rather than making assumptions.
 
-This means that you now have three key repositories to keep track of:
+- **Merging responsibilities**
 
-- **Upstream:** the main PrairieLearn repository on GitHub. You can't directly push to this, but this is where you will pull other people's commits from.
-- **Origin:** your forked repository on GitHub. You can push and pull directly to this.
-- **Local:** your clone of origin on your local computer. This is where you commit before pushing to origin.
+  - If a maintainer opens a PR, that person is responsible for merging it after it has been approved.
+  - For PRs from non-maintainers, the primary reviewer is responsible for merging the PR. If there are outstanding nitpicks or unresolved discussions, consult with the author before merging.
```

Comment:
Consider adding a note about trying to follow up on old PRs where possible? Things definitely get blocked / can't go in right away, but I think it's worth encouraging people to try and avoid their PRs becoming stale.

Reply:
I added a note in ce81502.